nio_command_block.py

```diff
@@ -74,15 +73,15 @@ def process_signals(self, signals):
                     resp = requests.get(url, headers=headers)
                     sigs = self._process_response(resp)
                     output_sigs.extend(sigs)
-            except Exception:
-                self._logger.exception('Failed to process signals')
+            except Exception as e:
+                self.logger.exception('Failed to process signals: %s' % e)
```

Comment:
Logging as an "exception" automatically includes the exception details so you don't want to also add it manually.